### ISSUE TITLE
No context menu entries at all, despite fiddling with save-in's GUI options

### ISSUE THREAD

**Issue by CottonEaster** [gyng/save-in#38]
Screenshot:

![image](https://user-images.githubusercontent.com/9998846/33401532-862216fe-d551-11e7-8bb6-0210219a30c9.png)

Irrespective of where I right-click, I never see any context-menu items from save-in.

My settings:
![image](https://user-images.githubusercontent.com/9998846/33401614-cc09ade4-d551-11e7-8d36-228e7f5a5a31.png)

Save-in version 1.6.
Firefox 57.0 (64-bit)
Windows (8.1 x64) and Linux Mint 18.3 (x64)

**Comment by gyng** [comment]
What are your path settings?

**Comment by CottonEaster** [comment]
Dear gyng

The path box shows (on Windows and Linux):

`.`

Since the path is defined as, 'Path relative to default download directory' and since my default download directory is, I believe, to my desktop (on both the Linux PC and the Windows one), I did not believe I needed to enter anything in the path box. Am I wrong?

As to the debug console: I'm afraid I'm unsure I've the energy for that.

**Comment by gyng** [comment]
If it's not showing up even when an image is clicked, then I really have no idea what's wrong.

`.` should save to the default downloads directory, but the context menu should still show up even regardless when an image is right-clicked.

**Comment by CottonEaster** [comment]
On Windows, the 'saveIn' context-menu option is now appearing when I right-click over images (and it can save those images - at least some of the time; trying to save e.g. the Reddit logo did produce the context-menu item but did not produce a file). However, I never see an option to save the whole page or to save as text.

**Comment by gyng** [comment]
It's a long shot, but did you save the settings?

Right now the settings page have to be manually saved by clicking the button right at the bottom. After some testing with people I found out nobody actually remembers to do so, so I changed it to save settings immediately in the next release.

If it still doesn't work you might want to wait a couple days until I release the next version, and see if it works then.
